packages/@aws-cdk/core/lib/aspect.ts

```diff
@@ -42,11 +42,11 @@ export class Aspects {
   }
 
   /**
-   * Adds an aspect to apply this scope before synthesis.
-   * @param aspect The aspect to add.
+   * Adds aspects to apply this scope before synthesis.
+   * @param aspect The aspects to add.
    */
-  public add(aspect: IAspect) {
-    this._aspects.push(aspect);
+  public add(...aspect: IAspect[]) {
```

Comment:
Relevant jsii output

```json
  {
      "docs": {
          "stability": "stable",
          "summary": "Adds aspects to apply this scope before synthesis."
      },
      "locationInModule": {
          "filename": "core/lib/aspect.ts",
          "line": 48
      },
      "name": "add",
      "parameters": [
          {
              "docs": {
                  "summary": "The aspects to add."
              },
              "name": "aspect",
              "type": {
                  "fqn": "aws-cdk-lib.IAspect"
              },
              "variadic": true
          }
      ],
      "variadic": true
  }

```

Reply:
Did some local testing with python, seems like `aspect` is treated like a `*arg` instead of a `**kwarg` so it breaks the scenario where the `aspect` parameter is explicitly defined.

Whether people are using it that way or not? Searching through the docs and python examples on GitHub suggest that people aren't using a named parameter here.

Pre-change
```python
## this works
Aspects.of(app).add(MyAspect())
## this works
Aspects.of(app).add(aspect=MyAspect())
```

post-change
```python
## this works
Aspects.of(app).add(MyAspect())
## this works
Aspects.of(app).add(MyAspect(),MyAspect())
## this does not work
Aspects.of(app).add(aspect=MyAspect())
```